test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -82,12 +83,15 @@ const (
 	extensionPointsLabelName = "extension_point"
 )
 
+var schedulerSubsystemAttemptDurationSeconds = metrics.SchedulerSubsystem + "_" + metrics.SchedulingAttemptDurationSeconds
+var metricLabelValues = []string{"Filter", "Score"}
```

Comment:
remove this, instead, use `metrics.Filter` and `metrics.Score` directly below.

Reply:
I have fixed this change in my new commit